src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -133,31 +136,41 @@ public static LogEventLevel FormDataLoggingLevel
         /// <param name="context">An <see cref="T:System.Web.HttpApplication"/> that provides access to the methods, properties, and events common to all application objects within an ASP.NET application </param>
         public void Init(HttpApplication context)
         {
-            context.LogRequest += LogRequest;
-            context.Error += Error;
-        }
+            context.BeginRequest += (o, args) =>
+            {
+                context.Context.Items[StopWatchKey] = Stopwatch.StartNew();
```

Comment:
May be better not to kick this off if the module is not enabled (and conditionally check again in EndRequest in case the key isn't present).